src/mgr/PyModuleRegistry.cc

```diff
@@ -46,14 +47,56 @@ void PyModuleRegistry::init()
 
   // Set up global python interpreter
 #define WCHAR(s) L ## #s
+#if PY_VERSION_HEX >= 0x03080000
+  PyConfig py_config;
+  PyConfig_InitIsolatedConfig(&py_config);
+  BOOST_SCOPE_EXIT_ALL(&py_config) {
+    PyConfig_Clear(&py_config);
+  };
+  // site_import is 1 by default, but set it explicitly as we do import
+  // site packages manually for Python < 3.8
```

Comment:
I am not sure I understand the comment. What's the relevance of what we do in the <3.8 case?

Reply:
i wanted to explain why i was adding a noop. the goal is to mirror the behavior of another branch which is picked by the preprocessor.